daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Pretty.scala

```diff
@@ -197,9 +197,13 @@ private[lf] object Pretty {
       case Node.Rollback(_) =>
         text("rollback")
       case create: Node.Create =>
-        "create" &: prettyContractInst(create.coinst)
+        intercalate(text(", "), create.signatories.map(p => text(p))) &
+          text("creates") &
```

Comment:
I think the only thing missing in this PR is getting these to agree with the number of parties, right?

Reply:
Yep, I'm working on aligning the scala part to haskell changes rn